### DISCUSSION TITLE
Outliner/Scenegraph Order

### DISCUSSION THREAD

**Discussion by ian898** [StjerneIdioten/I3D-Blender-Addon#167] [Ideas & Feature Requests]
I have a suggestion on sorting the scenegraph in both Blender and GE.

 if you added an alternative object name box like the alternative i3d mapping name you could add "number_" to object name to sort in blender objects in blender and have clean names on export.

**Comment by StjerneIdioten** [comment]
[This release](https://github.com/StjerneIdioten/I3D-Blender-Addon/releases/tag/v4.3.0) adds support for that.
